Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function (grunt) {
```

Comment:
Sounds good. I would also recommend testing on a simulated slow connection, since the system does need to be usable over, say, several users sharing one cellular connection. I've been using CharlesProxy set to "3G" throttling for my front end testing as a design goal; that seems to map fairly well to the actual speeds I get accessing the site over 3G.